Android/app/src/androidTest/java/PageObjects/TestAppPageObjects/HybridLocalAppPageObject.kt

```diff
@@ -37,15 +38,27 @@ import pageobjects.BasePageObject
 class HybridLocalAppPageObject(private val app: TestApplication) : BasePageObject() {
 
     fun assertAppLoads() {
-        Thread.sleep(timeout * 2)
+        val titleString = if (app.complexHybrid == "accounteditor") "Accounts" else "Contacts"
+        verifyInWebView(titleString)
+
+        if (app.complexHybrid == "accounteditor") {
+            verifyInWebView("New 0013u000011EMiVAAW Cached")
```

Comment:
Could we avoid hardcoded values?

Reply:
Unfortunately don't see how to avoid it.  Options for specifying ui elements in the webview are limited on Android:
![Screen Shot 2020-02-18 at 1 57 05 PM](https://user-images.githubusercontent.com/28540387/74781628-a78e3000-5256-11ea-9dc7-2ae46948fd51.png)

And XCode literally crashes if you attempt to obtain labels within the webview so I have no choice but to simply match strings.

Reply:
Could you create the record from the test with a unique name that you could then look for later in the test?
If that's too involved, could you pull the hardcoded values into some constants.

Reply:
Yeah, I was planning to at least put it into constants.  Adding a new account for each test would be better, but we are now up to ~12 unique test apps/scenarios per os (per os version) so I would like to keep login/data verification as simple as possible for maintenance.